src/animals/Antelope.js

```diff
@@ -0,0 +1,57 @@
+import { Animal } from '../organisms/Animal';
+import { getRandom } from '../utilities/getRandom';
+
+export class Antelope extends Animal {
+  constructor(positionY, positionX, age = 0) {
+    super(4, 4, positionY, positionX, age);
+  }
+
+  async action(board) {
+    // Double range of movement
+    const directions = [];
+    for (let directionY = -2; directionY <= 2; directionY++) {
+      for (let directionX = -2; directionX <= 2; directionX++) {
+        if (directionX === 0 && directionY === 0) continue;
+        directions.push({ positionY: directionY, positionX: directionX });
+      }
+    }
+
```

Comment:
This is really difficult to read. Avoid using `continue` if possible.
This should probably be a separate method.